docs/user/reporting/index.asciidoc

```diff
@@ -1,125 +1,166 @@
++
+NOTE: *Public URL* is available only when anonymous access is configured and your anonymous service account has privileges to access what you want to share.
 
+. Click *Copy link*.
 
 [float]
-[[full-page-pdf]]
-== Full page PDF layout &mdash;Canvas only
+[[download-as-json]]
+== Create a JSON file
 
-To create a PDF without margins surrounding the Canvas workpad, turn on *Full page layout* before generating the PDF.
+Create a JSON file for a workpad.
 
-[role="screenshot"]
-image::user/reporting/images/canvas-full-page-layout.png["Full Page Layout"]
+. Open the main menu, then click *Canvas*.
+
+. Open the workpad you want to share. 
+
+. From the toolbar, click *Share*, then select *Download as JSON*.
+
+[float]
+[[add-workpad-website]]
+== Share workpads on a website
+
+beta[] *Canvas* allows you to create _shareables_, which are workpads that you download and securely share on a website. 
+To customize the behavior of the workpad on your website, you can choose to autoplay the pages or hide the workpad toolbar.
+
+. Open the main menu, then click *Canvas*.
+
+. Open the workpad you want to share.
+
+. Click *Share > Share on a website*.
+
+. Follow the instructions.
+
+. To customize the workpad behavior to autoplay the pages or hide the toolbar, use the inline parameters.
++
+To make sure that your data remains secure, the data in the JSON file is not connected to {kib}. *Canvas* does not display elements that manipulate the data on the workpad.
++
+NOTE: Shareable workpads encode the current state of the workpad in a JSON file. When you make changes to the workpad, the changes do not appear in the shareable workpad on your website.
 
+. To change the settings, click the settings icon, then choose the settings you want to use.
 
 [float]
-[[manage-report-history]]
-== View and manage report history
+[[embed-code]]
+== Embed code
+
+Display your dashboard or visualization on an internal company website or personal web page with an iframe. Embedding other {kib} objects is generally supported, but you might need to manually craft the proper HTML code.
+
+Some users might not have access to the dashboard or visualization. For more information, refer to <<anonymous-access-and-embedding>> and <<embedded-content-authentication>>.
+
+. Open the main menu, then open the dashboard or visualization you want to share.
+
+. Click *Share > Embed code*. 
+
+. Specify how you want to generate the code:
+
+* To display only the current state, select *Snapshot*.
+
+* To display up-to-date changes, select *Saved object*.
+
+* Select the dashboard or visualization elements you want to include.
+
+* To generate a shortened link, select *Short URL*.
+
+* To automatically log in anonymous users when you have multiple authentication providers enabled, select *Public URL*.
++
+[role="screenshot"]
+image::images/embed-code-public-url.png[Embed code share menu with Public URL option highlighted]
```

Comment:
This image is blurry.

Reply:
I don't think I see what you see. This looks clear to me. 

![Screen Shot 2021-06-30 at 11 23 23 AM](https://user-images.githubusercontent.com/38990715/123997132-abc37500-d995-11eb-8fcb-3b2d2ca20756.png)